content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -677,15 +677,11 @@ data:
 
 ### Optional ConfigMaps
 
-In a Pod, or pod template, you can mark a reference to a ConfigMap as _optional_.
-If the ConfigMap is non-existent, the configuration for which it provides data in the Pod (e.g. environment variable, mounted volume) will be empty.
+You can mark a reference to a ConfigMap as _optional_ in a Pod specification.
+If the ConfigMap doesn't exist, the configuration for which it provides data in the Pod (e.g. environment variable, mounted volume) will be empty.
```

Comment:
A nit: we [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations.